src/Perpetuum/Services/Daytime/DayTime.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace Perpetuum.Services.Daytime
+{
+    public static class DayTime
+    {
+        private static readonly int _nltTotal = 1000;
+        //private static readonly int _sunrise = 150;
+        //private static readonly int _sunset = 750;
+        private static readonly double _nltDaySpan = TimeSpan.FromHours(3).TotalHours;
+        public static DayTimeInfo GetCurrentDayTime()
+        {
+            return GetNLTForTime(DateTime.Now);
+        }
+
+        public static DayTimeInfo GetNLTForTime(DateTime time)
```

Comment:
I don't like the use of the `NLT` acronym. `GetGameTime` or `GetLocalTime`. You don't need the `ForTime` suffix because you're passing a `DateTime` object in. A bit like saying `AddTwoIntegers(int first, int second)`, superfluous.

Reply:
Good point, "nian local time" is nlt in the game client, but that's arbitrary and not clear just looking at server code what that would mean